### PR TITLE
[WIP] Migrating to manifest v3

### PR DESCRIPTION
**Pull request by rajivharlalka**
- Updated the manifest to Version 3
- Removed unused scripts that caused import error.

### CODE REVIEW

**Comment by rajivharlalka** [comment]
Was getting an error in a script (skip-form.js) and neither was able to get which popup it asks for. 
image.

This is the only error left to be resolved in the migration.

**Comment by thealphadollar** [comment]
The popup has been removed from ERP, now. Please remove the script and its usage.

Thanks for the work on the migration.

File: skip-form.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 // Original Author:
 // https://github.com/rohitner/
 // Skips ERP annoying form that pops up
-window.setTimeout(() => {
-    document.getElementById("skiplink").click();
-}, 5000);
\ No newline at end of file
+// Pop Up removed from erp , commented for future Reference and Use
```
Comment (thealphadollar) [inline skip-form.js:4]: Do not leave a comment, and remove the code altogether. We will have the code in GitHub history for future reference.

**Review by thealphadollar — CHANGES_REQUESTED** [review]
Thanks, almost there.

**Review by thealphadollar — APPROVED** [review]

**Pull request merged**